voidcrew/code/modules/research/designs/machine_designs.dm

```diff
@@ -46,6 +46,14 @@
 	category = list("Misc. Machinery")
 	departmental_flags = DEPARTMENTAL_FLAG_SCIENCE | DEPARTMENTAL_FLAG_CARGO | DEPARTMENTAL_FLAG_ENGINEERING
 
+/datum/design/board/ammo_printer
+	name = "Machine Design (Ammo Printer Board)"
+	desc = "The circuit board for an Ammo Printer."
+	id = "ammoprinter"
```

Comment:
```suggestion
	id = "ammo_printer"
```

Reply:
wont this break the research node if its changed?

Reply:
> wont this break the research node if its changed?

the techweb node design id is ammo_printer, so both of them have to match for the circuit to be printable in R&D